containers/Ideas/IdeaCreate.js

```diff
@@ -110,6 +112,7 @@ export const IdeaCreate = props => {
       })
       //error handling is provisional and should probably go somewhere else eventually
       .then((response) => {
+        setSubmitPending(false)
```

Comment:
But to which one? We only do not need it in the first if as we go back to the idea list then, but for all other cases we would need it to be enabled again, so that the idea can be submitted again, right?

Reply:
yes

Reply:
it should stay disabled for the successful submit as user will be taken to different view